app/keepers/keepers.go

```diff
@@ -84,8 +84,18 @@ import (
 	"github.com/babylonlabs-io/babylon/v2/x/zoneconcierge"
 	zckeeper "github.com/babylonlabs-io/babylon/v2/x/zoneconcierge/keeper"
 	zctypes "github.com/babylonlabs-io/babylon/v2/x/zoneconcierge/types"
+	tokenfactorykeeper "github.com/strangelove-ventures/tokenfactory/x/tokenfactory/keeper"
+	tokenfactorytypes "github.com/strangelove-ventures/tokenfactory/x/tokenfactory/types"
 )
 
+var tokenFactoryCapabilities = []string{
```

Comment:
maybe lets add some doc explaining why those capabilites were chosen ? if those are defaults from some other docs we can add link to this doc here